trackers/node-tracker/src/tracker.ts

```diff
@@ -77,6 +93,8 @@ export function tracker(
   const addUserInformation = (payload: PayloadBuilder): void => {
     payload.add('duid', domainUserId);
     payload.add('nuid', networkUserId);
+    payload.add('sid', sessionId);
+    payload.add('vid', Number(sessionIndex));
```

Comment:
I will check this when I get home, but I think only string properties are allowed in the raw payload.

Reply:
You are right, it does end up a string!